pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   flutter:
     sdk: flutter
   audioplayers: ^6.0.0
+  battery_plus: ^5.0.2
```

Comment:
Hmm I think the idea is interesting but I don't think it's worth further increasing our bundle size for this. Right now the prompt shows after a timer which wouldn't have any dependency impact.

Reply:
This though would be shared anyway if #1202 would still be implemented. Same thoughts on that one as well?